src/javaharness/java/arcs/api/Handle.java

```diff
@@ -0,0 +1,19 @@
+package arcs.api;
+
+public abstract class Handle {
+  public StorageProxy storage;
```

Comment:
@cromwellian can we use AutoValue for these kinds of things? Perhaps for other classes as well.

Reply:
If the class has no logic and is just a holder, yes. If it has a bunch of logic other than field validation, I think it's frowned upon to use also value for this.

Reply:
there will be logic added in the following prs (see runtime/handle.ts)

Reply:
Don't worry about AutoValue for now, it just adds another dependency. :)